CONTRIBUTION.md

```diff
@@ -3,92 +3,106 @@
 
-Thanks for contributing!
+- [fetched from upstream](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository#fetching-changes-from-a-remote-repository)
+- [Rebased your changes against your root branch](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+- [Run the tests locally](docs/HACKING.md)
+- linted your changes with ```make lint```
+- ensured error messages are a single line 
+- updated relevant documentation if your PR changes user functionality  
 
-### Format of the pull request (PR)
-
-- The PR header for a feature should be prefixed with the Jira issue. Example: `WINC-123:`
-- The PR header for a bug should be prefixed with the Jira Bug identifier. Example: `OCPBUGS-123:`
-- Correctly prefixing the PR header will automatically associate the PR with the Jira issue or bug.
-  - In the case of bug fix PRs, this will also automatically transition the associated Jira bug.
-    - Opening the PR: `ASSIGNED` --> `POST`
-    - PR merges: `POST` --> `MODIFIED`
-    - `MODIFIED` to the `ON_QA` transition will have to be done manually by the PR author.
-    - PR closes without merging, the Jira bug transitions back to NEW status.
-- The individual commit messages should not be prefixed.
-
-### Format of the commit message
+### Format your commits 
+When making your changes, feel free to use whatever commit system you like, but when it’s time to open the pull request, be sure when rebasing that your commits follow the WMCO’s formatting requirements.
 
 We follow a convention for commit messages that is designed to answer two questions: what changed and why. The
 subject line should feature the what and the body of the commit should describe the why.
 
-The format can be described more formally as follows:
-
+A good commit message should look like 
 ```
 [subsystem] <what changed>
-<BLANK LINE>
+
 <why this change was made>
-<BLANK LINE>
-<footer>
+
+<Footer>
 ```
-Example for a sample feature:
+
+A real world example would look like 
 ```
 [docs] Add the Guidlines
 
-Cupcake ipsum dolor sit. Amet tart cheesecake tiramisu chocolate cake topping.
-Icing ice cream sweet roll. Biscuit dragée toffee wypas.
+adds the contribution guidelines for maintaining consistent style across the repo 
 
 Follow-up to Id5e7cbb1.
 ```
-
-The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
-lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
-git tools.
+The subject should be no longer than 50 characters, and the body should be no longer than 80 characters. There are some [githooks](https://github.com/jorisroovers/gitlint) you can install that can help enforce these style requirements.
 
 If it is a bug fix commit, the bug identifier should be mentioned in the commit message as `Fixes OCPBUGS-123` in a separate
```

Comment:
Is this still accurate? I thought we moved away from this since it messes with the OCPBUGS jira integration

Reply:
Correct, `Fixes OCPBUGS-123` should not be in the commit message.

Reply:
I thought what should not be doing is adding `OCPBUGS-123:` as prefix to the commit message. But if we should not be adding `Fixes` to the commit message but only to the PR message, then we should clearly call that out.